lldb/source/Plugins/Platform/gdb-server/PlatformRemoteGDBServer.cpp

```diff
@@ -684,6 +684,15 @@ Status PlatformRemoteGDBServer::RunShellCommand(
                                           signo_ptr, command_output, timeout);
 }
 
+bool PlatformRemoteGDBServer::CalculateMD5(const FileSpec &file_spec,
+                                           uint64_t &low, uint64_t &high) {
+  if (IsConnected()) {
+    // Note that high/low are switched in the gdb remote communication client
```

Comment:
It's good that you have a comment for this because it "feels" wrong. But why are `high/low` swapped in the gdb remote communication client in the first place? Is that intentional or is this working around a bug?

Reply:
Good point - I don't see a reason why it is swapped there. I'll change it so it's just consistent.